tests/testdata/buf.build/grpc/swift-protobuf/v1.0.0/eliza/plugin.sum

```diff
@@ -0,0 +1 @@
+h1:Ouy7fwcPAu9owIHJWt8sSknVb3j9zb27aNgesYMbWrE=
```

Comment:
took a peek at the generated code and looked right to me, but would be good to double-check.